[doc] fixed typos and punctuation (#29225)
Azure/azure-sdk-for-js: documentation/DPG-quickstart.md

3. **Edit rush.json**  
    

>     As the libraries in `azure-sdk-for-js` repository are managed by rush, you need to add an entry in `rush.json` under projects section for the first time to make sure it works. For example:

Reviewer: ```suggestion
    As the libraries in the `azure-sdk-for-js` repository are managed by rush, you need to add an entry in `rush.json` under projects section for the first time to make sure it works. For example:
```

    ```json
        {
          "packageName": "@azure/agrifood-farming",
          "projectFolder": "sdk/agrifood/agrifood-farming",
          "versionPolicyName": "client"
        },
    ```

>     Here, you also need to replace the `packageName`, `projectFolder` into your own services'.

Reviewer: ```suggestion
    You also need to replace the `packageName`, `projectFolder` entries with your own services'.
```
